### PR TITLE
update titlefont and icon border

### PR DESCRIPTION
**Pull request by ArshdeepSingh3-eaton**
Fixes 
- [422](https://github.com/etn-ccis/blui-react-native-component-library/issues/422) .
- [418](https://github.com/etn-ccis/blui-react-native-component-library/issues/418).

#### Changes proposed in this Pull Request:
- Updated css for title
- updated the icon border
- 

#### Screenshots / Screen Recording (if applicable)
-
Screenshot 2023-10-31 at 10 27 40 AM

#### To Test:
- 

#### Any specific feedback you are looking for?
-

### CODE REVIEW

File: components/src/md3/Hero/Hero.tsx [hunk 1]
```diff
@@ -119,6 +119,7 @@ export const Hero: React.FC<HeroProps> = (props) => {
                         backgroundColor: iconBackgroundColor || theme.colors.surface,
                         height: typeof iconSize === 'number' ? normalizeIconSize() : iconSize ?? 36 * fontScale,
                         width: typeof iconSize === 'number' ? normalizeIconSize() : iconSize ?? 36 * fontScale,
+                        borderRadius:25,
```
Comment (ektaghag-eaton) [inline components/src/md3/Hero/Hero.tsx:122]: use `borderRadius : 50` to match react component library. https://github.com/etn-ccis/blui-react-component-library/blob/dev/components/src/core/Hero/Hero.tsx#L76

Comment (ektaghag-eaton) [reply components/src/md3/Hero/Hero.tsx:122]: Can we move this style [here](https://github.com/etn-ccis/blui-react-native-component-library/blob/release/md3/components/src/md3/Hero/Hero.tsx#L26C9-L26C20)?

**Review by surajeaton — COMMENTED** [review]
prettier is failing. Please run `yarn precommit`

**Comment by ektaghag-eaton** [comment]
Test cases are failing
Screenshot 2023-11-01 at 11 19 22 AM

**Comment by surajeaton** [comment]
> Test cases are failing Screenshot 2023-11-01 at 11 19 22 AM

Tests are failing due to Drawer has `theme.colors.surfaceContainerLow`. Logged an issue https://github.com/etn-ccis/blui-react-native-component-library/issues/428

**Comment by surajeaton** [comment]
> > Test cases are failing Screenshot 2023-11-01 at 11 19 22 AM
> 
> Tests are failing due to Drawer has `theme.colors.surfaceContainerLow`. Logged an issue #428

But the snapshots should be updated.
Please run `yarn test -u`

**Comment by huayunh** [comment]
Were you able to make the background show conditionally? I think that's what we discussed the last time I was involved in this conversation.

By default:
* Hero component should not have a background color when rendered by itself
* When placed inside a scorecard, the Hero component shall pick up the scorecard's background color

Both of these shall be overridable by our adopter users

**Comment by huayunh** [comment]
Yeah, that would be the next best thing.

**Review by huayunh — COMMENTED** [review]
Please remember to update changelog and readmes

**Comment by ektaghag-eaton** [comment]
Screenshot 2023-11-06 at 11 58 42 AM

This is breaking, hero component is taking hero banner bg (see half upper space of hero component).

**Comment by ArshdeepSingh3-eaton** [comment]
Ekta that is supposed to happen as by default the hero has a transparent background
we need to provide background color to hero in order to show the background color

**Review by ektaghag-eaton — APPROVED** [review]

**Comment by huayunh** [comment]
I asked for both changelog and readme updates. I do not see the update in the changelog.

**Pull request merged**